spec/datetimeformat.html

```diff
@@ -303,6 +303,13 @@ <h1>PartitionDateTimePattern ( _dateTimeFormat_, _x_ )</h1>
             1. Else,
               1. Let _fv_ be an implementation and locale dependent String value representing `"ante meridiem"`.
             1. Add new part record { [[Type]]: `"dayPeriod"`, [[Value]]: _fv_ } as a new element of the list _result_.
+          1. Else if _p_ is equal to `"relatedYear"`, then
+            1. Let _v_ be _tm_.[[relatedYear]].
+            1. Let _fv_ be FormatNumber(_nf_, _v_).
+            1. Add new part record { [[Type]]: `"relatedYear"`, [[Value]]: _fv_ } as a new element of the list _result_.
+          1. Else if _p_ is equal to `"yearName"`, then
```

Comment:
same as above, should this be "year-name" instead?

Reply:
same answer